testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7401,6 +7470,87 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("safe_multiply(cast(3 as double), cast(null as bigint))");
   }
 
+  @Test void testSafeSubtractFunc() {
```

Comment:
JIRA Doc is always popular (not required).

Reply:
Done